stack-hie.yaml

```diff
@@ -0,0 +1,67 @@
+# This file was automatically generated by 'stack init'
```

Comment:
Is this file necessary? It broke my hls setup when I tried to fire up refinery.

Reply:
It prevents HLS from trashing builds/corrupting stuff, which it can do from time to time.